build.sbt

```diff
@@ -1046,6 +1060,8 @@ val graalNativeImageCommand = taskKey[String]("The native image command")
 val graalNativeImageOptions = settingKey[Seq[String]]("The native image options")
 val graalNativeImageClass = settingKey[String]("The class for the native image")
 val genNativeExecutable = taskKey[JPath]("Generate a native executable")
+val nativeInstallDirectory = taskKey[JPath]("The install directory for the native executable")
+val installNativeExecutable = inputKey[JPath]("Install the native executable")
```

Comment:
```suggestion
val installNativeExecutable = inputKey[JPath]("Install the native executable, for example by calling it as $ sbt installNativeExecutable ~/bin/")
```

Reply:
Sadly `Paths.get` doesn't work with `~/bin` but otherwise I like the suggestion with an absolute path.